mantidimaging/core/filters/crop_coords/crop_coords_gui.py

```diff
@@ -5,6 +5,5 @@
 
 # def _gui_register(main_window):
```

Comment:
Can we remove completely the commented out code?

Reply:
That is done in #117

Reply:
:+1: